matplotlib_scalebar/scalebar.py

```diff
@@ -251,6 +251,7 @@ def __init__(self, dx, units='m', dimension=SI_LENGTH, label=None,
         self.font_properties = font_properties
         self.fixed_value = fixed_value
         self.fixed_units = fixed_units
+        self.use_blit = use_blit
```

Comment:
Is there a reason not to called `self.set_animated(self.use_blit)` directly from the constructor?

Reply:
Yes, done!